### PR TITLE
Update to use latest component versions

### PR DESCRIPTION
**Pull request by xander-marjoram**

### CODE REVIEW

File: vanilla-app/main.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+import '@justeattakeaway/pie-card';
+import '@justeattakeaway/pie-cookie-banner';
 import '@justeattakeaway/pie-divider';
 import '@justeattakeaway/pie-icon-button';
-import '@justeattakeaway/pie-cookie-banner';
-import '@justeattakeaway/pie-card-container';
-import '@justeattakeaway/pie-toggle-switch';
-import '@justeattakeaway/pie-icons-webc/dist/icons/IconClose';
-import '@justeattakeaway/pie-icons-webc/dist/icons/IconSearch';
+import '@justeattakeaway/pie-modal';
```
Comment (ashleynolan) [inline vanilla-app/main.js:7]: 🎉

File: vanilla-app/package.json [hunk 2]
```diff
@@ -14,15 +14,20 @@
...
-    "@justeattakeaway/pie-button": "0.34.0",
-    "@justeattakeaway/pie-card-container": "0.11.0",
-    "@justeattakeaway/pie-cookie-banner": "0.7.0",
-    "@justeattakeaway/pie-css": "0.6.0",
-    "@justeattakeaway/pie-divider": "0.8.0",
-    "@justeattakeaway/pie-icon-button": "0.18.0",
-    "@justeattakeaway/pie-icons-webc": "0.6.0",
-    "@justeattakeaway/pie-modal": "0.28.0",
-    "@justeattakeaway/pie-toggle-switch": "0.14.0"
+    "@justeat/pie-design-tokens": "5.8.2",
```
Comment (jamieomaguire) [inline vanilla-app/package.json:17]: any reason not to use latest for this package?

File: vanilla-app/package.json [hunk 3]
```diff
@@ -14,15 +14,20 @@
...
+    "@justeattakeaway/pie-divider": "latest",
+    "@justeattakeaway/pie-icon-button": "latest",
+    "@justeattakeaway/pie-icons-webc": "latest",
+    "@justeattakeaway/pie-modal": "latest",
+    "@justeattakeaway/pie-switch": "latest",
+    "lit": "2.7.5"
+  },
+  "volta": {
+    "node": "18.18.2",
+    "yarn": "1.22.19"
```
Comment (jamieomaguire) [inline vanilla-app/package.json:31]: RE what @siggerzz mentioned, feel free to bump this to 3.5.0 if it's no overhead :)

**Review by jamieomaguire — APPROVED** [review]

File: vanilla-app/package.json (hunk 2 above)
Comment (xander-marjoram) [reply vanilla-app/package.json:17]: Short answer is no, there wasn't any reason 😄

**Review by ashleynolan — APPROVED** [review]

File: vanilla-app/package.json (hunk 3 above)
Comment (xander-marjoram) [reply vanilla-app/package.json:31]: It wasn't just a one-line change so I'll leave it for now

**Comment by xander-marjoram** [comment]
Not ready for merge yet because cookie banner still uses toggle-switch, so I'll wait until my other PR to update to switch has gone in.

**Pull request closed without merging**